### PR TITLE
✨😮Updating all SIG pages for Fall 2024

### PR DESCRIPTION
**Pull request by clee231**
## Description 

We have a ton of new SIGs! Let's get them on the website!

## Checklist

- [x] I've labeled the PR appropriately.
- [x] I've have built the website and verified that my changes work.
- [x] I've linked the appropriate issues and related PRs.

### CODE REVIEW

File: SIGs/Active/siggame.mdx [hunk 1]
```diff
@@ -27,14 +27,6 @@ Unity Hub can be installed [here](https://unity3d.com/get-unity/download)
 
 Github Desktop can be installed [here](https://unity3d.com/get-unity/download) (note: this is what we use to teach the basics of Github, but if you are more at home with another way of using Github such as Git Bash, that will work as well)
 
-## Contact
-
-You can reach either of the SIG leaders at their emails for additional information:
-
-- Zachary Robbins: zlr2 (at) uic.edu
```
Comment (lowpolyneko) [inline SIGs/Active/siggame.mdx:34]: Ethan's email is eky2@uic.edu if you wanna keep the contact info up-to-date.

File: SIGs/Active/sigsysadmin.mdx [hunk 2]
```diff
@@ -23,7 +23,7 @@ SIG SysAdmin is the system administration special interest group, the Spring 201
 - Chase Lee (Alumnus)
 - Daniel Stempniewicz (Alumnus)
 - Bharat Middha (Alumnus)
-- Christian Bingman
+- Christian Bingman (Alumnus)
 - Soham Gumaste
```
Comment (lowpolyneko) [inline SIGs/Active/sigsysadmin.mdx:27]: you should add meeeee ;)

**Review by lowpolyneko — CHANGES_REQUESTED** [review]
looks good to me otherwise :3

File: SIGs/Active/sigsysadmin.mdx (hunk 2 above)
Comment (clee231) [reply SIGs/Active/sigsysadmin.mdx:27]: Are you okay with your real name here? Or would you prefer another alias like your Github username or netid?

Comment (lowpolyneko) [reply SIGs/Active/sigsysadmin.mdx:27]: real name is fine haha, idrc with this account

**Review by bmiddha — APPROVED** [review]

**Review by lowpolyneko — APPROVED** [review]

**Comment by lowpolyneko** [comment]
i forget that i have write access to this repo sometimes... i applied my suggestions myself :D

**Pull request merged**